src/CommandHandler.cpp

```diff
@@ -453,39 +454,49 @@ void CommandHandler::handle(std::string const &Command) {
   LOG(Sev::Warning, "Could not understand this command: {}", Command);
 }
 
+std::string format_nested_exception(std::exception const &E,
+                                    std::stringstream &StrS, int Level) {
+  if (Level > 0) {
+    StrS << '\n';
+  }
+  StrS << fmt::format("{:{}}{}", "", 2 * Level, E.what());
+  try {
+    std::rethrow_if_nested(E);
+  } catch (std::exception const &E) {
+    format_nested_exception(E, StrS, Level + 1);
+  } catch (...) {
+  }
+  return StrS.str();
+}
+
+std::string format_nested_exception(std::exception const &E) {
+  std::stringstream StrS;
+  return format_nested_exception(E, StrS, 0);
+}
+
+static void dummy_emit_msg(std::string JobID, std::string Code,
```

Comment:
Is this useful?

Reply:
Are you sure that you're looking at the correct commits?

6d4f4a1 does not contain that function anymore.